extensions/ui/org.eclipse.smarthome.ui.paper/pom.xml

```diff
@@ -59,10 +59,10 @@
           <execution>
             <id>gulp build</id>
```

Comment:
ok, if it still is gulp after all, let's simply keep it - I do not really care much :-)